hw2/03-form.js

```diff
@@ -1,3 +1,24 @@
 /** Exercise 03 - Form **/
 
 // Add your code here
+
+function submitForm(){
+    console.log("========== Form Submission ===========");
```

Comment:
Nice use of bootstrap form styling. I made some of the same choices you did for those.

When running this code I noticed a few validation issues, though
- There is no validation to stops the user from submitting an empty form. The name and email fields are required, so some checking should be added to make sure they've been entered by the user. 
- There is no validation to check whether the user entered a valid email. I was able to enter random text as the email when I tested
- Additionally, I believe the requirements mentioned error messages when incorrect values are put in. So, I think it should give you feedback when you don't pass the validation. 
    - [This documentation article](https://getbootstrap.com/docs/5.0/forms/validation/) was really helpful for me and I was able to even directly borrow the example function for disabling the submit until all fields were filled out.

Lastly, another student said we were supposed to use `console.group()` for the output, but I'm not 100% sure if that's correct. I ended up implementing it anyway, as it seemed to be a useful way to group things and it matched the professor's output in the example, so that that as you will!